addons/partner_autocomplete/static/tests/partner_autocomplete_tests.js

```diff
@@ -4,8 +4,8 @@ odoo.define('partner_autocomplete.tests', function (require) {
     var FormView = require('web.FormView');
     var concurrency = require('web.concurrency');
     var testUtils = require("web.test_utils");
-    var AutocompleteField = require('partner.autocomplete.fieldchar');
-    var PartnerField = require('partner.autocomplete.many2one');
+//    var AutocompleteField = require('partner.autocomplete.fieldchar');
+//    var PartnerField = require('partner.autocomplete.many2one');
```

Comment:
LegacyCode ?

Reply:
I left the old tests as they were as I'm not done yet re-writing them.
They're skipped for now.